ProcessLib/GroundwaterFlow/CreateGroundwaterFlowProcess.cpp

```diff
@@ -57,6 +71,10 @@ std::unique_ptr<Process> createGroundwaterFlowProcess(
     auto media_map =
         MaterialPropertyLib::createMaterialSpatialDistributionMap(media, mesh);
 
+    DBUG("Check the media properties of GroundwaterFlow process ...");
+    checkMPLProperties(mesh, *media_map.get());
```

Comment:
Wouldn't this be sufficient?
```suggestion
    checkMPLProperties(mesh, *media_map);
```